packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -31,6 +34,9 @@ class LongPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
   /// Called when the pointer stops contacting the screen after the long-press gesture has been recognized.
   GestureLongPressUpCallback onLongPressUp;
 
+  /// Called when the pointer travels beyond[kTouchSlop] pixels from the original contact point after the long-press gesture has been recognized.
+  GestureLongPressRejectedCallback onLongPressRejected;
```

Comment:
Seems like a sensible addition. Though we shouldn't call it 'rejected' since 'rejected' means something specific in the gesture arena and we're precisely not rejected but accepted at this point. 

@HansMuller any suggestions for naming this?